wikidatarefisland/data_model/wikibase/value_types.py

```diff
@@ -76,7 +76,10 @@ def __eq__(self, other):
             date = isoparse(self.value[:4])
```

Comment:
I completely ignored the possibility of years longer than 4 digits when I wrote this, could you please add a ticket to support that (as obvs it is a possibility in wikidata...)

Reply:
Done! T254728